latentnetworks/vimure
# Add `install_vimure` to custom the version of the Python package

Closes #42 

How to validate?

- [ ] Clone this repository
- [ ] Checkout to `42-r-change-the-way-of-installing-python-dependencies`
- [ ] Remove all current installation of vimure (Bash `pip uninstall vimure`; R `remove.packages("vimure")`)
- [ ] Install vimure
```r
setwd("src/R")
devtools::install()
```
- [ ] Try to execute a vimure function
```r
gm_Multitensor(N=20, M=20)
```
You should see an error similar to:
```
Error: Valid installation of `vimure` not found.

Python environments searched for 'vimure' package:
 /home/gabriela-borges/anaconda3/envs/vimure/bin/python3.6
 /usr/bin/python3.8
 /home/gabriela-borges/anaconda3/bin/python3.8
 /home/gabriela-borges/anaconda3/envs/fastapi/bin/python3.9
 /home/gabriela-borges/anaconda3/envs/streamlit/bin/python3.9
 /home/gabriela-borges/anaconda3/envs/streamlit3/bin/python3.9

Python exception encountered:
 Traceback (most recent call last):
  File "/home/gabriela-borges/R/x86_64-pc-linux-gnu-library/4.1/reticulate/python/rpytools/loader.py", line 39, in _import_hook
    module = _import(
ModuleNotFoundError: No module named 'vimure'


You can install `vimure` using the install_vimure() function.
```
- [ ] Install vimure with `install_vimure()` and execute the same function
```r
install_vimure()
library(vimure)
gm_Multitensor(N=20, M=20)
```
- Typo errors 
- README.md files
